### ISSUE TITLE
No apparent way to change tooltip transparency system-wide

### ISSUE THREAD

**Issue by bhilmers** [lxqt/lxqt#2194]
Hello. I cannot find a way to remove transparency from tooltips. The tooltip transparency is system-wide, every place where hovered text exists. See image below:
![lubuntu-tooltip-transparency](https://user-images.githubusercontent.com/77466683/154582750-fdf86ef5-dcba-477b-969f-82a3b1a80a04.jpg)

OS: Ubuntu 20.04.3 LTS x86_64
Kernel: 5.15.15-76051515-generic
LXQt: 0.14.1
WM: Openbox
WM Theme: Lubuntu Arc

**Comment by tsujan** [comment]
Please don't open issues for asking questions. We have [Discussions](https://github.com/lxqt/lxqt/discussions).

I'd convert this to a discussion if it was related to LXQt. But it isn't.

Tooltip transparency may be produced by a QT/GTK theme (depending on the app) or by the compositor. Chheck your compositor settings.

**Comment by tsujan** [comment]
Yes, Discussions can be used for that purpose. This place is the bug tracker.

Most probably, the compositor you use has given all tooltips translucency. LXQt doesn't provide any compositor but any can be used with it. Find its settings and see if that's the case.

Ubuntu's forums are the best place for asking Ubuntu questions. LXQt isn't related to Ubuntu or any other distro.

**Comment by tsujan** [comment]
BTW, when you want to open an issue, a page is shown that has an "LXQt Discussions" section. It explicitly says, "For general questions, problems and else."

**Comment by stefonarch** [comment]
I see that picom.conf allows setting it separately:
![schermata-02-18-08-12](https://user-images.githubusercontent.com/10681413/154635756-fe829724-f375-46e9-bd67-a722bb1c9698.png)

**Comment by bhilmers** [comment]
> I see that picom.conf allows setting it separately...

Yes, this is very helpful stefonarch, thank you so much! Lubuntu 20 does not have Kwin, but rather has a tool named compton-conf. This tool doesn't have as many settings in the GUI, however, in ~/.config/compton.conf there is an entry for tooltips. I changed the opacity there and it works perfectly.

`wintypes : 
{
  tooltip : 
  {
    fade = true;
    shadow = false;
    opacity = 1.0;
    focus = true;
  };
};`

**Comment by stefonarch** [comment]
Nice to know. 
Picom-conf (fork of compton-conf) isn't released yet,  compton is deprecated in favour of picom - but still does his job.
